portfolio/src/main/java/com/google/sps/data/CommentsData.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import static com.googlecode.objectify.ObjectifyService.ofy;
+
+import java.util.List;
+import com.google.sps.data.Comment;
+
+/** Class that allows to perform operations to the comments in the database. */
+public class CommentsData {
+  /**
+   * Saves the comment in the database.
+   */
+  public static void save(Comment comment) {
+    ofy().save().entity(comment).now();
```

Comment:
I'm not sure how error handling in Objectify works but I assume this can throw some exceptions which would currently make your request return an standard 500 HTTP status and error page if storing fails for some reason. This seems ok for now (as it's really hard to recover form errors like this automatically) but might be something you can read up on a bit if interested. It can be as simple as trying out how to create a custom 404 and 500 page.

Reply:
I will have a look.